Dockerfile

```diff
@@ -54,7 +54,6 @@ ARG VITE_ORIGIN
 ARG VITE_ADMIN_TICKET_VALIDATION_URL
 ARG VITE_API_URI
 ARG VITE_CMS_URI
-ARG VITE_ENVIRONMENT
```

Comment:
We can keep this to indicate the real actual environment. The usage: https://dev.azure.com/City-of-Helsinki/kukkuu/_search?action=contents&text=VITE_ENVIRONMENT&type=code&lp=code-Project&filters=ProjectFilters%7Bkukkuu%7DRepositoryFilters%7Bkukkuu-pipelines%7D&pageSize=25&result=DefaultCollection/kukkuu/kukkuu-pipelines/GBmaster//devops/kukkuu-ui-build-config.yml

Reply:
This VITE_ENVIRONMENT is now in use in AppConfig, so it is needed here in build args too.